src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  API_KEY: "pub_3887e9c094a5e53a911088edca0a0a43a0e8"
```

Comment:
Obviously this is just a trivial exercise for a low risk third party account but an api key should never be submitted to the repo. This api key has the prefix of 'pub' so probably de-elevates the risks but I believe that 'shifting left' on security helps not do this. https://snyk.io/learn/shift-left-security/ and https://snyk.io/learn/secure-coding-practices/

Reply:
The second link talks about 'Secrets management'.

Reply:
@phasecurve , That's true ! It's even more unsafe for the fact that it is a public repository. It was a mistake in that commit. For subsequent commits, I ensured it is not there.

Reply:
You could roll your key if you want. Shouldn’t be a problem though.